### ISSUE TITLE
Broadcast topo setup issue

### ISSUE THREAD

**Issue by jiayisuse** [NVIDIA/nccl#590]
Hey team, recently we found NCCL broadcast with 8 RDMA NICs just utilizes one channel. The default NCCL on 8-NICs/host and 16 hosts setup ended up with 9 GB/s. When we enforce NCCL to use 1 channel by NCCL_MAX_NCHANNELS=1, the bandwidth is still 9 GB/s. NCCL version is 2.10.3. All other collectives seem good with multi-channel. Wondering if this broadcast behavior is as expected or not?

**Comment by sjeaugey** [comment]
That's not expected. Can you post the log with `NCCL_DEBUG=INFO` ?

**Comment by jiayisuse** [comment]
@sjeaugey Yes - https://gist.github.com/jiayisuse/3c7cca7a5e6da2210ddf8df2b5d3fbff. The bandwidth is very low as 15.05 GB/s

**Comment by sjeaugey** [comment]
Sorry I should I mentioned I wanted the full log, including the broadcast performance at the end. Also, it would be good to run with -b 8 -e 512M -f 2 instead of running on a single size.

So can you please run again  with `NCCL_DEBUG=INFO NCCL_DEBUG_SUBSYS=INIT,GRAPH`:
```
mpirun [...] -x NCCL_DEBUG=INFO -x NCCL_DEBUG_SUBSYS=INIT,GRAPH ./nccl_broadcast -b 8 -e 512M -f 2
```

**Comment by jiayisuse** [comment]
@sjeaugey The log is full log. You may need to click the 'view the full file' - https://gist.githubusercontent.com/jiayisuse/3c7cca7a5e6da2210ddf8df2b5d3fbff/raw/7f947e762119c20d3bf9c2555c7b23da63c01fc4/gistfile1.txt. The result is at the end.

Sure, let me run again and follow up with the latest result.

**Comment by stephenmsachs** [comment]
### Issue Cleanup: Helping Us Focus on Current Challenges

  We're reviewing older issues to ensure we prioritize the most relevant and active ones. Since this issue hasn't seen updates in over 6 months, we'll be closing it for now.

  ,*This change helps us focus our efforts on addressing any current issues our users are facing.* If this issue still affects you, please don't hesitate to reopen it with a quick update (e.g., "Still relevant on [version=X]").
  Thanks for your understanding and for contributing to NCCL.
